protocol/x/perpetuals/keeper/msg_server_update_perpetual_params_test.go

```diff
@@ -192,7 +193,19 @@ func TestUpdatePerpetualParams(t *testing.T) {
 				// Verify updated perpetual params in state.
 				updatedPerpetualInState, err := pc.PerpetualsKeeper.GetPerpetual(pc.Ctx, tc.msg.PerpetualParams.Id)
 				require.NoError(t, err)
-				require.Equal(t, tc.msg.PerpetualParams, updatedPerpetualInState.Params)
+				require.Equal(t, tc.msg.PerpetualParams.Ticker, updatedPerpetualInState.Params.Ticker)
```

Comment:
nit: Get initial perpetual and verify that the market type is the same as the initial.